sys/include/net/gnrc/gomach/gomach.h

```diff
@@ -85,6 +85,9 @@ extern "C" {
 #define GNRC_GOMACH_SUPERFRAME_DURATION_US        (300LU * US_PER_MS)
 #endif
 
+#ifndef RTT_FREQUENCY
+#warning "RTT_FREQUENCY undefined."
```

Comment:
I think we can make this an error. The define is clearly used in the division below.

Reply:
done.